services/addons/images/count_metric/README.md

```diff
@@ -36,8 +36,6 @@ To run this service, the following environment variables must be set:
 
 <b>SMTP_EMAIL:</b> The origin email that the count_metric will send the email from. This is usually associated with the SMTP server authentication.
 
-<b>SMTP_EMAIL_RECIPIENTS:</b> Recipient emails, delimited by ','.
```

Comment:
Lets add a little bit to the README for each service that is getting the email support to explain that the PostgreSQL table is a requirement.

Reply:
Added a note to the READMEs to explain that these services require the user_email_notification table to work correctly